test/t8_geometry/t8_geometry_implementations/t8_gtest_geometry_cad.cxx

```diff
@@ -42,7 +43,8 @@
 #include <TopoDS_Edge.hxx>
 #include <t8_element.h>
 #include <t8_cmesh_vtk_writer.h>
-#endif
+#include <t8_schemes/t8_default/t8_default_hex/t8_dhex.h>
+#endif /* T8_WITH_OCC */
```

Comment:
the t8code includes should not be behind the compiler macro for linkage with occ